user_guide_src/source/changelogs/v4.1.6.rst

```diff
@@ -0,0 +1,25 @@
+Version 4.1.6
```

Comment:
Include in toctree

Reply:
Thanks! Will do. This is what @kenjis was noting too.